tox.ini

```diff
@@ -18,6 +18,9 @@ deps =
     mako
     iniparse
     pyyaml
+    pykickstart
+    # required by pykickstart but not pulled in automatically :/
+    requests
```

Comment:
Odd. pykickstart's setup.py has `install_requires=['requests']` in it.

Reply:
Thanks for looking into this in https://github.com/pykickstart/pykickstart/issues/465 and https://github.com/pykickstart/pykickstart/pull/466 !